send-surveys/src/main/java/com/objectcomputing/pulsesurvey/send/surveys/SendSurveysController.java

```diff
@@ -13,13 +14,18 @@
 import java.util.Map;
 import java.util.HashMap;
 import java.lang.Math;
+
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Body;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Post;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@FunctionBean("send-surveys")
-public class SendSurveysFunction extends FunctionInitializer
-        implements Supplier<SendSurveys> {
-    private static final Logger LOG = LoggerFactory.getLogger(SendSurveysFunction.class);
+@Controller("/sendEmails")
+public class SendSurveysController {
```

Comment:
And let's just call this SurveysController...then make the root path "/surveys"

Reply:
done